projects/cdk/directives/auto-focus/autofocus.directive.ts

```diff
@@ -10,18 +10,30 @@ import {
 @Directive({
     selector: '[tuiAutoFocus]',
     providers: TUI_AUTOFOCUS_PROVIDERS,
+    exportAs: 'tuiAutoFocus',
 })
-export class TuiAutoFocusDirective implements AfterViewInit {
+export class TuiAutoFocusDirective<T extends HTMLElement = HTMLElement>
+    implements ElementRef<T>, AfterViewInit
+{
     @Input('tuiAutoFocus')
     autoFocus: boolean | '' = true;
 
     constructor(
+        @Inject(ElementRef) private readonly elementRef: ElementRef<T>,
         @Inject(TUI_AUTOFOCUS_HANDLER) private readonly handler: TuiAutofocusHandler,
     ) {}
 
+    get nativeElement(): T {
```

Comment:
I don't like expanding public API unnecessarily for the sake of one component. Let's add `read: ElementRef` counterparts in `InputCardGroup` if we need native elements there. `exportAs` also doesn't seem to be required.

Reply:
fixed